packages/neon/neon_notes/lib/widgets/notes_floating_action_button.dart

```diff
@@ -13,17 +13,18 @@ class NotesFloatingActionButton extends StatelessWidget {
   @override
   Widget build(final BuildContext context) => FloatingActionButton(
         onPressed: () async {
-          final result = await showDialog<List>(
+          final result = await showDialog<(String, String?)>(
```

Comment:
Is it possible to do something like `final (title, category)? = await showDialog<(String, String?)>(`? I really don't know, but it might make it a little cleaner

Reply:
I tried to but because of the nullability it's not working.
I mean how would the if check look like?

The only thing we could do it use `result.$1` and `result.$2` instead of assigning them into different variables (We only use them once)

Reply:
I think assigning to variables makes the code easier to understand, so better leave it as is